Update index.qmd to describe dataset reporting (#4)
reichlab/flusight-dashboard: pages/index.qmd

[View The Latest Forecasts](forecast.html){.btn .btn-primary role="button"}

> The target data for the FluSight challenge is hospitalizations with influenza as reported to the National Healthcare Safety Network (NHSN); this data is publicly available through the CDC's [Weekly Hospital Respiratory Data (HRD) Metrics](https://data.cdc.gov/Public-Health-Surveillance/Weekly-Hospital-Respiratory-Data-HRD-Metrics-by-Ju/mpgq-jmmr/about_data) dataset. That dataset page provides the following important note about hospitalization reporting which may impact interpretation of these forecasts:
> 
> > Data for reporting dates through April 30, 2024 represent data reported during a previous mandated reporting period as specified by the HHS Secretary. Data for reporting dates May 1, 2024 – October 31, 2024 represent voluntarily reported data in the absence of a mandate. Data for reporting dates beginning November 1, 2024 represent data reported during a current mandated reporting period.

Reviewer: I would suggest to use a callout instead of a block quote. 

```suggestion
The target data for the FluSight challenge is hospitalizations with influenza as reported to the National Healthcare Safety Network (NHSN); this data is publicly available through the CDC's [Weekly Hospital Respiratory Data (HRD) Metrics](https://data.cdc.gov/Public-Health-Surveillance/Weekly-Hospital-Respiratory-Data-HRD-Metrics-by-Ju/mpgq-jmmr/about_data) dataset.

::::{.callout-note}

Data for reporting dates through April 30, 2024 represent data reported during a previous mandated reporting period as specified by the HHS Secretary. Data for reporting dates May 1, 2024--October 31, 2024 represent voluntarily reported data in the absence of a mandate. Data for reporting dates beginning November 1, 2024 represent data reported during a current mandated reporting period.

::::

```
